sound/soc/sof/ipc4-pcm.c

```diff
@@ -56,16 +56,40 @@ sof_ipc4_sps_to_time_info(struct snd_sof_pcm_stream *sps)
 	return stream_priv->time_info;
 }
 
+static
+char *sof_ipc4_set_multi_pipeline_state_debug(struct snd_sof_dev *sdev, char *buf, size_t size,
+					      struct ipc4_pipeline_set_state_data *trigger_list)
+{
+	int i, offset = 0;
+
+	for (i = 0; i < trigger_list->count; i++) {
+		offset =+ snprintf(buf + offset, size - offset, " %d",
+				   trigger_list->pipeline_instance_ids[i]);
+
+		if (offset >= size - 1) {
+			buf[size - 1] = '\0';
+			break;
+		}
+	}
+	return buf;
+}
+
 static int sof_ipc4_set_multi_pipeline_state(struct snd_sof_dev *sdev, u32 state,
 					     struct ipc4_pipeline_set_state_data *trigger_list)
 {
 	struct sof_ipc4_msg msg = {{ 0 }};
 	u32 primary, ipc_size;
+	char debug_buf[32];
 
 	/* trigger a single pipeline */
-	if (trigger_list->count == 1)
+	if (trigger_list->count == 1) {
 		return sof_ipc4_set_pipeline_state(sdev, trigger_list->pipeline_instance_ids[0],
 						   state);
+	}
+
+	dev_dbg(sdev->dev, "pipeline instances%s",
```

Comment:
@jsarha, can you rebase this and drop this patch?
We have this already, but in a cleaner way: f2f08c0471d5d

Reply:
Oh I was hasty. I do not know how this ended up here. I must have rebased on some other branch than I intended to. Probably a local version of topic/sof-dev, instead of origin/topic/sof-dev. I'll drop the commit.